File: transpile/giada/swingjs/layout/FlowLayout.js
/**
 * the java.awt.FlowLayout clone
 *
 * @author gianpiero.diblasi
 */
class FlowLayout extends LayoutManager {

  static  LEFT = 0;

  static  CENTER = 1;

  static  RIGHT = 2;

  static  LEADING = 3;

   align = 0;

   hGap = 0;

   vGap = 0;

  constructor(align, hGap, vGap) {
    super("flowlayout");
    this.align = align;
    this.hGap = hGap;
    this.vGap = vGap;
  }
}

File: src/giada/awtjs/FlowLayout.java
package giada.awtjs;

/**
 * the java.awt.FlowLayout clone
 *
 * @author gianpiero.diblasi
 */
public class FlowLayout extends LayoutManager {

  public static final int LEFT = 0;
  public static final int CENTER = 1;
  public static final int RIGHT = 2;
  public static final int LEADING = 3;
  public static final int TRAILING = 4;

  public final int align;
  public final int hGap;
  public final int vGap;

  public FlowLayout(int align, int hGap, int vGap) {
    super("flowlayout");

    this.align = align;
    this.hGap = hGap;
    this.vGap = vGap;
  }
}
